src/persistence.js

```diff
@@ -356,7 +356,7 @@ export default function _Persistence(mpInstance) {
         }
 
         if (!mpInstance._Store.SDKConfig.useCookieStorage) {
-            localStorageData.l = mpInstance._Store.isLoggedIn ? 1 : 0;
+            localStorageData.l = mpInstance._Store.isLoggedIn; // ? 1 : 0;
```

Comment:
this changes what is in persistence actually.  it saves `true` instead of `1/0` and that's because the 1:0 translation should be happening inside of the encodePersistence, similar to what is happening when the key === `ie`. 

<img width="1302" alt="image" src="https://github.com/mParticle/mparticle-web-sdk/assets/5377436/b3ac5963-4c89-47e1-b4d3-232a2d1de741">

Please revert this, or add this to the encoding layer.